src/pumps/GeoEmaAndCumSmaPump.sol

```diff
@@ -126,7 +126,8 @@ contract GeoEmaAndCumSmaPump is IPump, IInstantaneousPump, ICumulativePump {
         bytes16[] memory byteReserves = new bytes16[](reserves.length);
 
         // Skip {_capReserve} since we have no prior reference
-        for (uint i = 0; i < reserves.length; i++) {
+        uint length = reserves.length;
```

Comment:
can you put this assignment above line 126 so that bytesReserve can also use length?

Reply:
yes no doubt